src/sonic-yang-models/yang-models/sonic-logger.yang

```diff
@@ -0,0 +1,77 @@
+module sonic-logger{
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-logger";
```

Comment:
can we update the namespace to sonic-net ? Thanks.

Reply:
Let`s do it for all yang models later in one go.